frontend/src/screens/apps/AppCreated.tsx

```diff
@@ -61,20 +61,24 @@ function AppCreatedInternal() {
     }
     // dispatch a success event which can be listened to by the opener or by the app that embedded the webview
     // this gives those apps the chance to know the user has enabled the connection
-    const nwcEvent = new CustomEvent("nwc:success", { detail: {} });
+    const nwcEvent = new CustomEvent("nwc:success", {
```

Comment:
do we actually consume this custom event anywhere? 

I know `window.opener.postMessage` is consumed by Alby JS SDK. Do we need both?

Reply:
Ah, maybe this one is for a native app (I need to test it)